modules/system/rolls/test-wfrp4e.js

```diff
@@ -535,7 +535,13 @@ export default class TestWFRP {
       chatOptions.sound = undefined;
 
     //this.result.other = this.result.other.join("<br>")
-    this.result.other = this.preData.other.join("<br>")
+
+    if (this.preData.other.length > 0) {
+      this.result.other = this.preData.other.join("<br>");
+    }
+    else if(this.result.other && this.result.other.join) {
+      this.result.other = this.result.other.join("<br>");
+    }
```

Comment:
Not sure what these two cases are for? Couldn't it just be
```js
if (this.result.other instanceof Array)
{
   this.result.other = this.result.other.join("<br>")
}
```

Reply:
Sorry for that, it was quick and dirty during session, so i forgot to fix it before commit.

Reply:
Now it still has these two cases (if/else)

```js
    if (this.preData.other.length > 0) {
      this.result.other = this.preData.other.join("<br>");
    } 
    else if (this.result.other instanceof Array) {
       this.result.other = this.result.other.join("<br>")
    }
```

But as my previous comment suggests, can't it be one if statement testing if it's an array?

Reply:
Bumping this as I'd like to merge soon

Reply:
i would have to try recreate the issue one more time, but i noticed again, that in some cases, the preData.other is empty, and result.other is already an Array and not string. thats why two statements. it is something that "works for me"

Reply:
Well, as it is, it will throw an error if `this.preData.other` is a string, because `join` is not a string function. 

If we need two cases though, I think it's simpler and cleaner to combine them using `&&` conditional

Reply:
Ok, i've made another commit with proposal to get rid of preData.other as it was introduced purely for unofficialgirmoire. I hope this could solve the problem in clean way.

Reply:
Well this PR is already way out of scope for just being fixes, can you remove all changes related to `other` and put that into another PR?

Reply:
ok, changes are reverted and separate PR has been created for that